python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -5114,15 +5114,16 @@ def test_df_astype_to_categorical_ordered(ordered):
 
 
 @pytest.mark.parametrize(
-    "dtype,args",
-    [(dtype, {}) for dtype in ALL_TYPES]
-    + [("category", {"ordered": True}), ("category", {"ordered": False})],
+    "dtype",
+    [dtype for dtype in ALL_TYPES]
+    + [
+        cudf.CategoricalDtype(ordered=True),
+        cudf.CategoricalDtype(ordered=False),
+    ],
 )
-def test_empty_df_astype(dtype, args):
+def test_empty_df_astype(dtype):
     df = cudf.DataFrame()
-    kwargs = {}
-    kwargs.update(args)
-    assert_eq(df, df.astype(dtype=dtype, **kwargs))
+    assert_eq(df, df.astype(dtype=dtype))
```

Comment:
Should this be a pandas comparison instead? 

`assert_eq(df.to_pandas().astype(dtype=dtype), df.astype(dtype=dtype))`

Reply:
I'm not sure, but it doesn't hurt to add this comparison too.